frontend/packages/github/src/index.ts

```diff
@@ -2,3 +2,5 @@ export * from './api.browser'
 export * from './api.server'
 export * from './config'
 export * from './types'
+
+export { getRepositoryBranches } from './api.server'
```

Comment:
Is this necessary? Maybe included in line 2

Reply:
Thank you for noticing. I have corrected it.🙏
https://github.com/liam-hq/liam/pull/1019/commits/ecabf91fddc59a86c3defadc2b77b60f926dfba1